pkg/config/config.go

```diff
@@ -34,19 +37,87 @@ type IgnoreEntry struct {
 	Reason      string    `toml:"reason"`
 }
 
+type IgnorePackageVersionEntry struct {
+	Name        string    `toml:"name"`
+	Version     string    `toml:"version"`
+	Ecosystem   string    `toml:"ecosystem"`
+	IgnoreUntil time.Time `toml:"ignoreUntil"`
+	Reason      string    `toml:"reason"`
+}
+
+type OverridePackageVersionLicenseEntry struct {
+	Name            string   `toml:"name"`
+	Major           string   `toml:"major"`
+	Minor           string   `toml:"minor"`
+	Patch           string   `toml:"patch"`
+	ExactVersion    string   `toml:"exactVersion"`
```

Comment:
Nit: Is there any reason to qualify this with `exact` in `exactVersion` ? IMO `version` on its own is pretty clear.

Reply:
now that we're merging the two, version seems fine to me.